### PR TITLE
Add support for tuple returning from pages

### LINKED ISSUE #492: Add support for tuple returning from pages.
When a page returns a tuple, unwrap the tuple as the args for a container element (such as pc.box). This would allow for syntax such as
```
def index():
  return pc.text("Hello"), pc.text("World")
```

### PR DESCRIPTION
**Pull request by danik292**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired change?

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: reflex/app.py [hunk 1]
```diff
@@ -329,6 +330,12 @@ def add_page(
         # Wrap the component in a fragment.
         component = Fragment.create(component)
 
+        # If component is not a Component object, try to unwrap it into a Fragment
+        if not isinstance(component, Component):
```
Comment (masenf) [inline reflex/app.py:334]: if the line above wraps `component` in a fragment, then this condition will never be true.

File: reflex/app.py [hunk 2]
```diff
@@ -329,6 +330,12 @@ def add_page(
...
+                with suppress(TypeError):
+                component = Fragment.create(*[i for i in component])
```
Comment (masenf) [inline reflex/app.py:336]:
```suggestion
                component = Fragment.create(*component)
```

no need for the extra list.

File: reflex/app.py (hunk 1 above)
Comment (danik292) [reply reflex/app.py:334]: So I can delete it?

File: reflex/app.py (hunk 2 above)
Comment (danik292) [reply reflex/app.py:336]: Ok

File: reflex/app.py (hunk 1 above)
Comment (danik292) [reply reflex/app.py:334]: @masenf I can delete it so

Comment (danik292) [reply reflex/app.py:334]: I give it a try

**Comment by danik292** [comment]
@masenf can you take a look on this?

**Comment by danik292** [comment]
@masenf @Alek99 pls look on this one

**Comment by masenf** [comment]
> pls look on this one

@danik292 i see three blank lines and a duplicate `import` statement

**Comment by danik292** [comment]
> > pls look on this one
> 
> @danik292 i see three blank lines and a duplicate `import` statement

I think you (@masenf ) implement it so iI can close it

**Comment by masenf** [comment]
I added `overlay_component`, which caused conflict; the code still doesn't support tuple of `Component`

Maybe you could start off by writing a test case that attempts to `add_page` with a function that returns tuple and then write the code necessary to make it pass.

**Comment by danik292** [comment]
I start over with a new PR becauce i need to use my main breach and  use your commited versino of code

**Pull request closed without merging**